metadata-schema/closureSchemaOpen.schema.json

```diff
@@ -0,0 +1,71 @@
+{
+  "$id": "/schema/open-closure",
+  "type": "object",
+  "allOf": [
+    {
+      "if": {
+        "properties": {
+          "closure_type": {
+            "enum": [
+              "open_on_transfer",
```

Comment:
for now only support Open - there is a card to restrict this in the base as well